docs/userGuide/siteConfiguration.md

```diff
@@ -152,6 +159,10 @@ The following properties will apply to `index.md`:
 
 **An array of external scripts to be referenced on all pages.** To reference an external script only on specific pages, `externalScripts` should be specified in `pages` instead. Scripts referenced will be run before the layout script.
 
+#### **`globalOverride`**
+
+**Globally overrides properties in the front matter of all pages.** Any property included in the global override will automatically be merged with the front matter of every single page, and override them if the property exists. Also overrides front matter properties specified in `pages`.
```

Comment:
This documented behaviour is no longer true? `frontMatter` now has a higher overriding priority than `globalOverride` with the latest implementation.

Also update `frontMatter`'s description accordingly.